### ISSUE TITLE
[Accessibility][On Focus - Create from scratch]: Tab focus does not jump within the 'Define conversation objective' dialog box automatically after clicking enter on ‘Next’ button.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2015]
**User Experience:**
Keyboard dependent users would face difficulty if focus does not move to the dialog box that opens up after selecting any control.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Navigate on homepage with tab key up to 'New' button and press enter on it.
3. 'Create from scratch' pop up will open.
4. Select any radio button and press enter on 'Next' button.
5. Observe the issue with tab focus. 

**Actual Result:**
Tab focus does not jump within the 'Define conversation objective' dialog box automatically after clicking enter on ‘Next’ button. User has to press tab key, then focus comes within the 'Name' edit field.

**Expected Result:**
Tab focus should automatically jump within the 'Define conversation objective' dialog box after clicking enter on ‘Next’ button.

**Note:** 
With screen reader as soon as user click on 'Next' button, screen reader start reading the content of homepage. Then, on pressing tab key, screen reader focus comes within the 'Name' edit field and read the dialog information: (refer Attachment).

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

**Attachment:**
[MAS3.2.1_Create from Scratch_Next.zip](https://github.com/microsoft/BotFramework-Composer/files/4224230/MAS3.2.1_Create.from.Scratch_Next.zip)
[MAS3.2.1_Create from Scratch_NexT_Screen reader.zip](https://github.com/microsoft/BotFramework-Composer/files/4224231/MAS3.2.1_Create.from.Scratch_NexT_Screen.reader.zip)

**Comment by ashish315** [comment]
@corinagum As checked now the screen reader reads the info of 'Name' edit field along with the location information. The narrator should announce the info about the pane name as well as the bot name.

**Comment by beyackle** [comment]
Verified this is still a little awkward. Hitting Next will correctly place focus in the "Name" field in the creation dialog, but it doesn't read the introductory text. It does, for some reason, read the file path (from the Location input below), which it shouldn't.

**Comment by liweitian** [comment]
> Verified this is still a little awkward. Hitting Next will correctly place focus in the "Name" field in the creation dialog, but it doesn't read the introductory text. It does, for some reason, read the file path (from the Location input below), which it shouldn't.

I found the issue there. I am working on it.

**Comment by liweitian** [comment]
Hi @beyackle, which screen reader are you using? I used windows 10 narrator. Some people online said it is just a 'toy'. So I check this issue again with NVDA and JAWS. It works as expected.

**Comment by beyackle** [comment]
I've been using Windows 10 Narrator. I wouldn't dismiss it as just a "toy"; it's something customers use too.

**Comment by beyackle** [comment]
On advice from @cwhitten - since this works in NVDA, we're not going to worry about this one for now.

**Comment by ashish315** [comment]
As checked the issue is resolved so, closing the issue.
